plugin-management-library/src/main/java/io/jenkins/tools/pluginmanager/impl/PluginManager.java

```diff
@@ -83,8 +80,8 @@ public PluginManager(Config cfg) {
 
 
     public void start() {
-        if (!refDir.mkdir()) {
-            System.out.println("Unable to create plugin directory");
+        if (!refDir.exists() && !refDir.mkdir()) {
```

Comment:
In new code it is better to use https://docs.oracle.com/javase/7/docs/api/java/nio/file/Files.html for new code so that you use APIs which throws exceptions instead of boolean returns